api/data/pipeline.py

```diff
@@ -79,15 +80,32 @@ def main():
     # Merging Clinvar
     clinvar = clinvar_data.copy()[["Name(clinvar)",
                                    "Germline classification(clinvar)",
-                                   "Accession(clinvar)"]]
+                                   "Accession(clinvar)",
+                                   "GRCh38Location(clinvar)"]]
+
     clinvar["VariantOnTranscript/DNA"] = (clinvar["Name(clinvar)"].
                                           apply(from_clinvar_name_to_cdna_position))
-
     main_frame = pd.merge(main_frame,
                           clinvar,
                           how="outer",
                           on=["VariantOnTranscript/DNA"]).drop("Name(clinvar)", axis=1)
 
+    start_merge = pd.merge(main_frame,
+                           clinvar,
+                           how="outer",
+                           left_on="position_g_start",
+                           right_on="GRCh38Location(clinvar)").drop(["Name(clinvar)", "GRCh38Location(clinvar)"],
```

Comment:
regarding `(clinvar)` extenstion. I know you used convention according to this file, but I would like to ask you to change it to underscore according to lovd columns naming style.